vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -52,6 +53,48 @@ void ReactImage::OnLoadEnd(winrt::event_token const &token) noexcept {
   m_onLoadEndEvent.remove(token);
 }
 
+void ReactImage::ResizeMode(react::uwp::ResizeMode value) {
+  if (m_resizeMode != value) {
+    m_resizeMode = value;
+
+    bool switchBrushes{false};
+
+    if (m_useCompositionBrush != ShouldUseCompositionBrush()) {
+      switchBrushes = true;
+      m_useCompositionBrush = ShouldUseCompositionBrush();
+    }
+
+    if (switchBrushes) {
+      SetBackground(m_memoryStream != nullptr, false);
+    } else {
+      const auto bitmapBrush{Background().as<winrt::ImageBrush>()};
+      bitmapBrush.Stretch(ResizeModeToStretch(m_resizeMode));
+    }
+  }
+}
+
+bool ReactImage::ShouldUseCompositionBrush() {
+  return m_resizeMode == ResizeMode::Repeat;
+}
+
+winrt::Stretch ReactImage::ResizeModeToStretch(react::uwp::ResizeMode value) {
+  switch (value) {
+    case ResizeMode::Cover:
+      return winrt::Stretch::UniformToFill;
+    case ResizeMode::Stretch:
+      return winrt::Stretch::Fill;
+    case ResizeMode::Contain:
+      return winrt::Stretch::Uniform;
+    default: // ResizeMode::Center
```

Comment:
>default: // ResizeMode::Center [](start = 4, length = 30)

For safety and clarity, please add an assert that value is not ResizeMode::Repeat, along with a comment that we should never be calling this function for that case (since it's handled by the Should UseCompositionBrush()) code path.